app/Filament/Resources/ServerResource/Pages/EditServer.php

```diff
@@ -62,6 +73,26 @@ public function form(Form $form): Form
                         Tab::make('Information')
                             ->icon('tabler-info-circle')
                             ->schema([
+                                Forms\Components\ToggleButtons::make('condition')
+                                    ->label('Status')
+                                    ->formatStateUsing(fn (Server $server) => $server->condition)
+                                    ->options(fn ($state) => collect(array_merge(ContainerStatus::cases(), ServerState::cases()))
+                                        ->filter(fn ($condition) => $condition->value === $state)
+                                        ->mapWithKeys(fn ($state) => [$state->value => str($state->value)->replace('_', ' ')->ucwords()])
+                                    )
+                                    ->colors(collect(array_merge(ContainerStatus::cases(), ServerState::cases()))->mapWithKeys(
+                                        fn ($status) => [$status->value => $status->color()]
+                                    ))
+                                    ->icons(collect(array_merge(ContainerStatus::cases(), ServerState::cases()))->mapWithKeys(
+                                        fn ($status) => [$status->value => $status->icon()]
+                                    ))
+                                    ->columnSpan([
+                                        'default' => 2,
+                                        'sm' => 1,
+                                        'md' => 1,
+                                        'lg' => 1,
+                                    ]),
+
```

Comment:
Status is already there:
![grafik](https://github.com/user-attachments/assets/b0352654-38ff-4905-b231-6e633957b8a4)

```suggestion

```